bin/stacks/rpc-gateway-dashboard.ts

```diff
@@ -196,6 +196,51 @@ function getCheckHealthMetricsForChain(chainId: ChainId) {
   return metrics
 }
 
+function getRpcGatewayQuoteCountForChain(chainId: ChainId) {
+  const metrics = []
+  metrics.push([
+    'Uniswap',
+    `RPC_GATEWAY_GET_QUOTE_REQUESTED_CHAINID: ${chainId}`,
```

Comment:
If you were to add the `RPC_GATEWAY` metrics and dashboard, you will need to add RPC gateway specific alarms as well.

Reply:
Do you mean adding alarm to the `RPC_GATEWAY_GET_QUOTE_500_CHAINID: xxx`? The total amount of quote requests going through RPC gateway doesn't need to trigger alarm for now.

Reply:
Well, the alert should be resilient to the request volume. The alerts by default "treat missing data as missing", and therefore it's common to see alerts in the insufficient data state:

![Screenshot 2024-04-02 at 7.21.57 PM.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/BB54fKe6Y10GvrrdKXQN/6b53df98-2aeb-4548-b4a8-8aebc84ca9a5.png)

If we see the value in introducing the `RPC_GATEWAY_GET_QUOTE_500_CHAINID: xxx` metric, then I think it's also the case to introduce the corresponding alerts.

Reply:
Thanks. I can make a follow up PR to add the alert.